### DISCUSSION TITLE
Restoring (X) Close Tab on Vertical Tabs

### DISCUSSION THREAD

**Discussion by oskibear2012** [soulhotel/FF-ULTIMA#33] [Ideas]
Is this possible to do with the given tab styles? I don't want to use Sidebery, etc.

**Comment by Kaedriz** [comment]
Done slight test, and it could be doable I think, @soulhotel would need to check.
`margin-left: 4px` seems good to hide it during retracted vertical bar, then `overlay-x: clip` on `.tab-content` seems to fix horizontal scroll issue appearing from this change. There is some CSS conflict too I think, because only active tab seems to show close button, then after a while the rest.

**Comment by DD1TTY** [comment]
Eagerly anticipating having a close tab x button in the vertical tab sidebar! Until then, I'm just using Ctrl + W to close the currently active tab
